api/swagger.yaml

```diff
@@ -2102,6 +2102,16 @@ definitions:
                     SpreadDescriptor:
                       description: "label descriptor, such as engine.labels.az"
                       type: "string"
+          Platforms:
+            description: "An array of supported platforms."
+            type: "array"
+            items:
+              type: "object"
+              properties:
+                Architecture:
```

Comment:
The OCI platform struct seems identical to manifestlist.PlatformSpec, which is what this is meant to consume.

Should we do this for 17.07, given that the filter isn't going to change right now?